docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>Why do we set <code>normalize=False</code>? Did they do that in the original paper?</p><p> </p><p>Why are we looking for the discord index first? It seems like the wrong order of steps. I feel like the first thing that somebody would do is to:</p><p> </p><ol><li>Apply <code>stump</code> to the entire time series</li><li>Then they realize that the "interesting" subsequence pair isn't in their top motif(s)</li><li>Finally, we explain why a naive approach is insufficient to find the "interesting" subsequence pair</li></ol><p> </p><p>Then, we can motivate the question, "How would/could we find the "interesting" subsequence pair?" and then provide clear and concrete steps on how to do it. Otherwise, this section seems out of place or presented too soon.</p><p> </p><p>I think we have to think about "what is the simplest thing that the user would have tried?" and then, when that fails, we build upon that knowledge and move past it with some better suggestion(s)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>